update check-interface-throughput-snmp rule

bmadhavi74 opened the pull request.

gvarao commented on community_supplied/Interface/snmp-interface-throughput.rule:
```diff
@@ -1,11 +1,282 @@
+                    then {
+                        status {
+                            color red;
+                            message "$interface-name input throughput utilization ($in-bw-util %) crossed threshold value ($threshold %). Mbps:$in-mbps Bps:$input-bytes bps";
```
it (Bps:$input-bytes bps) is bytes not bps hence same. cane be removed

bmadhavi74 replied:
Addressed and changed the message from "Bps:$input-bytes bps" to "Input-bytes:$input-bytes"

gvarao commented on community_supplied/Interface/snmp-interface-throughput.rule:
```diff
@@ -1,11 +1,282 @@
+                    then {
+                        status {
+                            color green;
+                            message "$interface-name input throughput utilization ($in-bw-util %) is normal. Mbps:$in-mbps Bps:$input-bytes bps ";
```
it (Bps:$input-bytes bps) is bytes not bps hence same. cane be removed

bmadhavi74 replied:
Addressed and changed the message from "Bps:$input-bytes bps" to "Input-bytes:$input-bytes"

gvarao commented on community_supplied/Interface/snmp-interface-throughput.rule:
```diff
@@ -1,11 +1,282 @@
+                    then {
+                        status {
+                            color red;
+                            message "$interface-name output throughput utilization($out-bw-util%) crossed threshold value ($threshold %). Mbps:$out-mbps Bps:$output-bytes bps";
```
it (Bps:$output-bytes bps) is bytes not bps hence same. cane be removed

bmadhavi74 replied:
Addressed and changed the message from "Bps:$output-bytes bps" to "Output-bytes:$output-bytes"

gvarao commented on community_supplied/Interface/snmp-interface-throughput.rule:
```diff
@@ -1,11 +1,282 @@
+                    then {
+                        status {
+                            color green;
+                            message "$interface-name output throughput utilization ($out-bw-util %) is normal. Mbps:$out-mbps Bps:$output-bytes bps";
```
it (Bps:$output-bytes bps) is bytes not bps hence same. cane be removed

bmadhavi74 replied:
Addressed and changed the message from "Bps:$output-bytes bps" to "Output-bytes:$output-bytes"

gvarao commented on community_supplied/Interface/snmp-interface-throughput.rule:
```diff
@@ -1,11 +1,282 @@
+            */
+            variable interface-name-variable {
+                value .*;
+                description "Interfaces to be monitored";
```
s/Interfaces to be monitored/Input interfaces to be monitored in regex/g

bmadhavi74 replied:
Changed "Interfaces to be monitored" to "Enter interface name in regex i.e. ge-.*"

gvarao commented on community_supplied/Interface/snmp-interface-throughput.rule:
```diff
@@ -1,11 +1,282 @@
+                description "Interfaces SNMP sensor to data from network device";
+                snmp {
+                    table IF-MIB::ifXTable;
+                    frequency 60s;
```
Increase frequency from 60s to 180s

bmadhavi74 replied:
Addressed and Increased frequency from 60s to 180s

bmadhavi74 commented:
- Added Dynamic threshold fields for both input and output bw utilization (dt-in-bw-util) and (dt-out-bw-util) respectively.

- Added yellow(medium) trigger handling the dynamic thresholds.
